### DISCUSSION TITLE
WebGL compositing still slow on Linux/Mac Firefox?

### DISCUSSION THREAD

**Discussion by elliottslaughter** [emilk/egui#2440] [Q&A] [answered]
The eframe README currently says:

> On Linux and Mac, Firefox will copy the WebGL render target from GPU, to CPU and then back again (https://bugzilla.mozilla.org/show_bug.cgi?id=1010527#c0), slowing down egui.

But the bug was closed 1 year ago with "RESOLVED FIXED" status.

So is this still the case on these platforms?

**Comment by emilk** [comment, marked as answer]
Good catch - `eframe` works great in Firefox these days so I'll remove that text!
